<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="charLayout">
         <item>
          <widget class="QLabel" name="charLabel">
           <property name="text">
            <string>Character Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="charName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="raceLayout">
         <item>
          <widget class="QLabel" name="raceLabel">
           <property name="text">
            <string>Race:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="race"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="bgLayout">
         <item>
          <widget class="QLabel" name="bgLabel">
           <property name="text">
            <string>Background:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bgs"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="npcBox">
         <property name="text">
          <string>NPC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="classLayout">
         <item>
          <widget class="QLabel" name="classLabel">
           <property name="text">
            <string>Class:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="className"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="levelLayout">
         <item>
          <widget class="QLabel" name="levelLabel">
           <property name="text">
            <string>Level:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="level">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="uidLayout">
         <item>
          <widget class="QLabel" name="uidLabel">
           <property name="text">
            <string>ID # (random is fine):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="uid">
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="statsLayout">
       <item>
        <widget class="QLabel" name="statsLabel">
         <property name="text">
          <string>Stats:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="strLayout">
           <item>
            <widget class="QSpinBox" name="str">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="strLabel">
             <property name="text">
              <string>Strength</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="dexLayout">
           <item>
            <widget class="QSpinBox" name="dex">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dexLabel">
             <property name="text">
              <string>Dexterity</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="conLayout">
           <item>
            <widget class="QSpinBox" name="con">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="conLabel">
             <property name="text">
              <string>Constitution</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="intLayout">
           <item>
            <widget class="QSpinBox" name="inte">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="intLabel">
             <property name="text">
              <string>Intelligence</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="wisLayout">
           <item>
            <widget class="QSpinBox" name="wis">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="wisLabel">
             <property name="text">
              <string>Wisdom</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="chaLayout">
           <item>
            <widget class="QSpinBox" name="cha">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="chaLabel">
             <property name="text">
              <string>Charisma</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
 </slots>
</ui>
